### PR TITLE
Feature/desktop menu

### PR DESCRIPTION
**Pull request by AwdamnKaya**
Added a desktop and mobile navigational menu with very basic css.

### CODE REVIEW

**Review by Alexander-Lindstroem — APPROVED** [review]
LGTM

**Review by DimitarIM — CHANGES_REQUESTED** [review]
The menu breaks down after 770px for me. Needs responsiveness to be implemented
Screenshot 2024-11-29 153044

**Review by DimitarIM — APPROVED** [review]

File: html/birds.html [hunk 1]
```diff
@@ -7,40 +7,29 @@
     <link rel="stylesheet" href="../css/sidebar.css">
     <link rel="stylesheet" href="../css/footer.css">
     <link rel="stylesheet" href="../css/general.css">
-    <link rel="stylesheet" href="../css/birds.css">
-    <script src="../scripts/sidebar.js" defer></script>
-    <script src="../scripts/birds.js" defer></script>
-    <title>Home</title>
+    <link rel="stylesheet" href="../css/menu.css">
+    <title>Birds</title>
```
Comment (Alibaba000101) [inline html/birds.html]: nice

**Review by Alibaba000101 — APPROVED** [review]
Nice work chief

**Review by Alexander-Lindstroem — APPROVED** [review]
LGTM

**Pull request merged**